ibm_catalog.json

```diff
@@ -15,14 +15,14 @@
+			"short_description": "Add your IBM Cloud account or enterprise to IBM Cloudability to start analyzing and tracking all of your cloud expenses.",
+			"long_description": "Easily add your IBM Cloud account(s) to your IBM Cloudability account with this deployable architecture (DA). All of the necessary steps to add your IBM Cloud billing data to an existing IBM Cloudability account are managed for you using infrastructure as code (IaC). This includes the following:\n\n1. Creating a Key Protect encrypted Object Storage bucket to store your billing reports.\n2. Enabling IBM Cloud [billing report exports](/docs/account?topic=account-exporting-your-usage&interface=ui) to receive billing reports updated daily.\n3. Grant IBM Cloudability access to read the reports from the bucket\n4. Add your IBM Cloud account to Cloudability.\n5. Verify that the IBM Cloud integration with Cloudability is successful.\n\nYour IBM Cloud billing data will begin showing up in IBM Cloudability the next day. \nIf you don't have an IBM Cloudability account then start by creating your [free trial](https://www.apptio.com/products/cloudability/#cldy-trial)\n. If you already have access to IBM Cloudability and are ready to add your IBM Cloud account then start by [creating your access credentials](/docs/track-spend-with-cloudability?topic=track-spend-with-cloudability-planning) and then [deploying with projects](/docs/track-spend-with-cloudability?topic=track-spend-with-cloudability-deploy-cloud). \n\nFor more details refer to the [Cloudability Enablement deployment guide](/docs/track-spend-with-cloudability) for more information.",
 			"provider_name": "IBM",
 			"offering_docs_url": "https://cloud.ibm.com/docs/track-spend-with-cloudability",
 			"offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cloudability-onboarding/refs/heads/main/images/light-theme.svg",
 			"features": [
 				{
-					"title": "Automatic updates of your IBM Cloud billing reports in a Object Storage Bucket",
+					"title": "Store your latest IBM Cloud billing reports in a Cloud Object Storage",
```

Comment:
typo: Store your latest IBM Cloud billing reports in ~~**a**~~ Cloud Object Storage